### PR TITLE
DHE-351 Service deliveries

### PR DESCRIPTION
**Pull request by bmcorser**

### CODE REVIEW

**Comment by uktitools** [comment]
Can one of the admins verify this patch?

File: test/unit/test_etl.py [hunk 1]
```diff
@@ -67,29 +89,41 @@
...
+    assert result == ODATA_SERVICE_DELIVERY_INPUT_DATA
+
+
+def test_odata_to_django_service_delivery():
+    result = transform.odata_to_django('ServiceDeliverySet', ODATA_SERVICE_DELIVERY_INPUT_DATA)
+
+    django_data = DJANGO_SERVICE_DELIVERY_DATA.copy()
+    assert result == django_data
+
+
```
Comment (bmcorser) [inline test/unit/test_etl.py]: I often enjoy to use [`@pytest.mark.parametrize`](http://doc.pytest.org/en/latest/parametrize.html#pytest-mark-parametrize-parametrizing-test-functions), maybe you will too.

File: korben/bau/webserver.py [hunk 2]
```diff
@@ -1,25 +1,20 @@
...
-from . import views
 
 
 DEFAULT_SETTINGS = {
-    'odata_metadata': lambda: db.get_odata_metadata(),
-    'django_metadata': lambda: db.get_django_metadata(),
-    'cdms_client': lambda: api.CDMSRestApi()
+    'odata_metadata': db.get_odata_metadata,
+    'django_metadata': db.get_django_metadata,
+    'cdms_client': api.CDMSRestApi
```
Comment (bmcorser) [inline korben/bau/webserver.py]: 😊

File: korben/etl/spec.py [hunk 3]
```diff
@@ -42,6 +42,19 @@
...
+            ('optevia_Service', (('Id', 'service_id'),),),
+            ('optevia_ServiceDeliveryStatus', (('Id', 'status_id'),),),
+            ('OwningTeam', (('Id', 'owning_team_id'),),),
+            ('optevia_UKRegion', (('Id', 'uk_region_id'),),),
+            ('optevia_Sector', (('Id', 'sector_id'),),),
+            ('optevia_LeadCountry', (('Id', 'lead_country_id'),),),
+            ('optevia_ServiceOffer', (('Id', 'service_offer_id'),),),
+            ('optevia_ServiceProvider', (('Id', 'service_provider_id'),),),
+        ),
+    },
```
Comment (bmcorser) [inline korben/etl/spec.py]: We will add the `local` fields today.

**Comment by bmcorser** [comment]
Rebased to bring flake8 in

**Comment by bmcorser** [comment]
Please note, the merge of this PR should be coordinated with the merge of https://github.com/uktrade/data-hub-leeloo/pull/104

**Pull request merged**